### PR TITLE
feat(issue-136): add language support

### PR DESCRIPTION
**Pull request by jisung-lee7**
## Issue Link

[issue-136](https://github.com/kafelix496/coding-for-fun/issues/136)

## Notes

- Need to change css(motion?) when hover dropdown items. (just mouse pointer, need to change finger?)

### CODE REVIEW

**Comment by jisung-lee7** [comment]
Please don't approve it yet, I'm still looking at it.

File: package.json [hunk 1]
```diff
@@ -41,7 +41,8 @@
     "react-markdown": "^9.0.1",
     "rehype-raw": "^7.0.0",
     "sass": "^1.77.6",
-    "tailwindcss-animate": "^1.0.7"
+    "tailwindcss-animate": "^1.0.7",
+    "universal-cookie": "^7.1.4"
```
Comment (jiyeol-lee) [inline package.json:45]: remove this one

File: src/contexts/dictionary-provider/dictionary-provider.tsx [hunk 2]
```diff
@@ -25,12 +27,12 @@ export const DictionaryProvider: FC<DictionaryProviderProps> = ({
 }
 
 export const useDictionary = () => {
-  const dictionary = useContext(DictionaryContext)
+  const xxx = useContext(DictionaryContext)
```
Comment (jiyeol-lee) [inline src/contexts/dictionary-provider/dictionary-provider.tsx:30]: why `xxx`?

File: src/contexts/dictionary-provider/utils.ts [hunk 3]
```diff
@@ -55,9 +55,10 @@ export const translate = <
   const targetDictionaryValue = findTargetDictionaryValue(key, dictionary)
 
   if (targetDictionaryValue === undefined) {
-    throw new Error(
-      `Cannot find dictionary value for key: ${key}. You must've ignored typescript error.`
-    )
+    return ''
+    // throw new Error(
+    //   `Cannot find dictionary value for key: ${key}. You must've ignored typescript error.`
+    // )
```
Comment (jiyeol-lee) [inline src/contexts/dictionary-provider/utils.ts:61]: Remove this comment

**Review by jiyeol-lee — CHANGES_REQUESTED** [review]

File: src/contexts/dictionary-provider/dictionary-provider.tsx (hunk 2 above)
Comment (jisung-lee7) [reply src/contexts/dictionary-provider/dictionary-provider.tsx:30]: At that time, I temporarily changed it because I was studying and I forgot.

**Review by jiyeol-lee — APPROVED** [review]
Good job

**Pull request merged**